terraform/azure.tf

```diff
@@ -589,3 +589,20 @@ resource "azurerm_virtual_machine_extension" "virtual_machine_extension" {
     }
 SETTINGS
 }
+
+resource "azurerm_sql_server" "sql-server-1" {
+  name                          = "${var.sql-server-name}"
+  resource_group_name           = "${azurerm_resource_group.rg.name}"
+  location                      = "${var.location}"
+  version                       = "12.0"
```

Comment:
Any value in this being a variable?  I was bitten recently by a hard-coded OS version going EOL but that's a different story ;)

Reply:
👍